index.d.ts

```diff
@@ -5,7 +5,7 @@
 /// <reference types="mocha" />
 
 declare type BN = import("bn.js");
-declare type Web3 = import("web3");
+declare type Web3 = import("web3").default;
```

Comment:
are u sure about this change? it breaks compilation for me. Please make sure that you local deps are up to date

Reply:
My compiler is complaining without `.default`. I'm using the latest `web3` at `1.0.0-beta.55` and `@types/web3` at `1.0.18`. Might be a glitch with my `tsconfig` tho... Care to share yours please?

This is the message I get on this import:

```
Module 'web3' does not refer to a type, but is used as a type here. Did you mean 'typeof import('web3')'?
```

Reply:
https://github.com/ethereum-ts/truffle-typechain-example check out here. I just updated dependency there